cluster_management/src/main/java/com/pinterest/rocksplicator/publisher/ZkBasedPerResourceShardMapPublisher.java

```diff
@@ -285,6 +284,9 @@ private void removeResourceFromZkGZIPCompressedPerResourceShardMap(
             continue;
           }
 
+          // First remove the local cache
+          this.latestResourceToConfigMap.remove(resourceName);
```

Comment:
This is a bug fix... since we have a local per resource shardmap... we have to ensure that it is also part of cleanup process...